core/controllers/topics_and_skills_dashboard.py

```diff
@@ -117,6 +123,40 @@ def get(self) -> None:
         for topic_summary_dict in topic_summary_dicts:
             topic_summary_dict['classroom'] = topic_classroom_dict.get(
                 topic_summary_dict['id'], None)
+            topic = topic_fetchers.get_topic_by_id(topic_summary_dict['id'])
```

Comment:
Do not put get_by_id() in a for loop. Use a single get_multi() call instead, otherwise it will take too long in production.

Reply:
Done